src/main/resources/logback.xml

```diff
@@ -1,35 +1,11 @@
 <configuration>
-
-    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%date %level [%thread] %logger{0} %msg%n</pattern>
-        </encoder>
-    </appender>
-
-    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>nro-stats.log</file>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%date %level [%thread] %logger{0} %msg%n</pattern>
-        </encoder>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>nro-stats.%d{yyyy-MM-dd}.log</fileNamePattern>
-            <maxHistory>60</maxHistory>
-        </rollingPolicy>
-    </appender>
-
-    <appender name="ERRORFILE" class="ch.qos.logback.core.FileAppender">
-        <file>nro-stats.err</file>
```

Comment:
Wait this might be relied upon by Icinga btw. If this is not empty it will alert 247.

Reply:
Correct. We'll not update the old instances, but only will use this in pipelines output.